en/lessons/gravity-model.md

```diff
@@ -0,0 +1,807 @@
+
+```
+setwd(PATH) #change "PATH" to the full location on your computer where the files can be found
+
+```
+
+Notice that line 4 is the line that solves the equation for us, using the [glm.nb](https://stat.ethz.ch/R-manual/R-devel/library/MASS/html/glm.nb.html) function, which is short for "generalized linear model - negative binomial". This line requires a number of inputs:
+
+* our variables using the column headers as written in the .csv file, along with any logging that must be done to them (vagrants, log(population), log(distance), wheat, wages, wageTrajectory). If you were running a model with your own data, you would adjust these to reflect your column headers in your dataset.
+* where the code can find the data - in this case a variable we've defined in line 3 called `gravityModelData`.
+
+The outputs of the calculation can be seen in Figure 9:
+
+{% include figure.html filename="figure9.png" caption="Figure 9: The summary of the above code, showing the weightings for each variable and the y-intercept, listed under the 'Estimate' heading ($\beta_{0}$ to $\beta_{5}$. This summary also shows a number of other calculations, including [statistical significance](https://en.wikipedia.org/wiki/Statistical_significance)." %}
+
+## Step 3: Calculating the Estimates for Each County
+
+Because we now have the y-intercept ($β0$), the weightings ($β1-5$), and the 5 variable values for each county ($P$, $d$, $Wh$, $Wa$, $WaT$), we have all the numbers we need to solve for the model's predicted value for a county: the final result.
+
+We have to do this once for each of the 32 counties.
+
+You could do this with a scientific calculator, by creating a spreadsheet formula, or writing a computer programme. To do this automatically in *R*, you can add the following to your code and re-run the programme. This `for` loop calculates the expected number of vagrants from each of the 32 counties in the example and prints the results for you to see:
+
+```
+for (entry in c(gravityModelData$County)){
+  print(paste("Result for County ", gravityModelData$County[entry],
+              (exp(-3.848
+                  + (1.235 * log(gravityModelData$population[entry]))
+                  + (-0.542 * log(gravityModelData$distance[entry]))
+                  + (-0.024 * gravityModelData$wheat[entry])
+                  + (-0.025 * gravityModelData$wages[entry])
+                  + (-0.014 * gravityModelData$wageTrajectory[entry])
+              ))
+        ))
+}
+```
+
+To build understanding, I suggest doing one county long-hand. This tutorial will use Hertfordshire as the long-hand example (but the process is exactly the same for the other 31 counties).
+
+Using the data for Hertfordshire in Table 3, and the weightings for each variable in Table 4, we can now complete our formula, which will give the result of 95:
+
+```
+estimated vagrants =
```

Comment:
I'm not sure what to make of the formatting here - do you intend it to be pseudo code marked off with backticks? Or LaTeX? You can't just add comments to rendered LaTeX like this, so I marked it up as code instead. My two cents: since you're actually operationalizing the equations above, putting in real numbers now, this should all be marked as code you run in R.